src/pages/Index.vue

```diff
@@ -1013,6 +1060,10 @@
 			color         : $colorBgFooterPrimary;
 			margin-top    : 3rem;
 			margin-bottom : 10rem;
+
+			@media screen and (max-width:700px){
+				margin: auto;
```

Comment:
The logo needs a bigger margin there, perhaps not as big as when in desktop mode, but not 0